aleph0/src/createActions.ts

```diff
@@ -1,18 +1,35 @@
 import { z } from "zod";
+import kebabCase from "lodash/kebabCase";
 import { loadSnippets } from "./lib/utils";
 import { createFile, createFolder, readFile } from "./lib/file";
-import { Technology, zodTechnology } from "./types";
+import { SnippetFile, Technology } from "./types";
 import { RunnableFunctionWithParse } from "openai/lib/RunnableFunction.mjs";
 
-export const createActions = (
+export const createActions = async (
   technology: Technology,
   projectRoot: string
-): Record<string, RunnableFunctionWithParse<any>> => {
+): Promise<Record<string, RunnableFunctionWithParse<any>>> => {
+  const { snippets: snippetsWithoutIds } = await loadSnippets(technology);
+  const files: Record<string, SnippetFile> = {};
+
+  const snippets = snippetsWithoutIds.map((snippet) => {
```

Comment:
I would move this to a function with an explainable name. Maybe also add a typescript definition of snippet files with an id.

Reply:
1. I don't see a major benefit of extracting every 3 lines into their own function. I'd rather just read code down the screen than have to jump across functions. I added a comment now in any case. Feel free to extract to a function but to me feels like minimal value and if anything makes it less readable.
2. `add a typescript definition` - I don't see how this helps. I'm all for TypeScript protection but as this is only used in one place there's no need to create its own type. The type is inferred here and there are no errors.

Reply:
It's all preference for readability. No worries, no need to change.